{
  Throwable err=assertThrows(log,new Callable(){
    @Override public Object call() throws Exception {
      return mgr.putIfAbsent(parentId,fileName,fileInfo);
    }
  }
,IgniteCheckedException.class,msg);
  assertTrue("Unexpected cause: " + err.getCause(),err.getCause() instanceof IgfsException);
}
